### ISSUE TITLE
AVT 1 - Overflow Menu w/link had DAP violations when disabled knob is selected

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4533]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component and select the disable knob
Run DAP and the following violations are present (see screenshot):
<img width="917" alt="Screen Shot 2019-11-01 at 3 24 59 PM" src="https://user-images.githubusercontent.com/21676914/68054426-40a33600-fcbc-11e9-9e53-a01f2b464897.png">

**Comment by dakahn** [comment]
As discussed offline disabled components are exempt from 1.4.3's contrast requirements. Looks like an error with DAP 👍
